packages/client/src/pages/SearchResult/SearchResult.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import './SearchResult.scss';
+import TrecipeCard from '../../components/TrecipeCard/TrecipeCard';
+import { RootState } from '../../redux';
+import { connect } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+import { Link, withRouter, RouteComponentProps } from 'react-router-dom';
+import { fetchResultRequest } from '../../redux/SearchResult/action';
+import Trecipe from '../../../../shared/models/trecipe';
+import Destination from '../../../../shared/models/destination';
+import { DestinationCard } from '../Map/DestinationCard/DestinationCard';
+
+type SearchResultProps = ReturnType<typeof mapStateToProps> &
+    ReturnType<typeof mapDispatchToProps> &
+    RouteComponentProps<{ searchKey: string }>;
+class SearchResult extends React.Component<SearchResultProps, {}> {
+    componentDidMount(): void {
+        let searchKey = this.props.location.search;
+        console.log('1');
+        console.log(searchKey);
+        searchKey = searchKey.substring(3);
+        console.log('2');
+        console.log(searchKey);
+        this.props.fetchResultRequest(searchKey);
+    }
+    render() {
+        console.log('here');
+        return (
+            <div className="search-result-wrapper">
```

Comment:
Some how husky hook is broken right now, so make sure to run `yarn lint` before every commit

Reply:
OK